### ISSUE TITLE
NIghtly failure in `files_external_s3`

### ISSUE THREAD

**Issue by SagarGi** [owncloud/files_external_s3#1017]
### Description
This app is failing on nightly build with: 
```
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
- Required (in require-dev) package "roave/security-advisories" is not present in the lock file.
This usually happens when composer files are incorrectly merged or the composer.json file is manually edited.
Read more about correctly resolving merge conflicts https://getcomposer.org/doc/articles/resolving-merge-conflicts.md
and prefer using the "require" command over editing the composer.json file directly https://getcomposer.org/doc/03-cli.md#require-r
make: *** [Makefile:153: vendor/bamarni/composer-bin-plugin] Error 4
```

Build Link: https://drone.owncloud.com/owncloud/files_external_s3/2389/1/2

**Comment by phil-davis** [comment]
Should be fixed by #1018 
This issue can be closed if nightly CI passes tonight.

**Comment by SagarGi** [comment]
Nightly is passing so closing this issue.
